src/components/wiki-documentation/create-article-form.tsx

```diff
@@ -371,11 +375,12 @@ const CreateOrEditArticleForm = ({
               rows={3}
               value={description}
               onInput={handleDescriptionChange}
-              label={`${strings.wikiDocumentation.labelDescription} *`}
+              label={strings.wikiDocumentation.labelDescription}
+              required
             />
           </Grid>
         </Grid>
-        <RichTextEditorLexical ref={editorRef} markdownContent={article?.content || ""}/>
+        <RichTextEditorLexical ref={editorRef} markdownContent={article?.content || "This need to be filled"}/>
```

Comment:
Localize.
I would suggest for the english something like:
"Article content is required".